python/packages/autogen-core/docs/src/_static/custom.css

```diff
@@ -2,6 +2,10 @@
   font-size: 0.8rem;
 }
 
+html {
+  --pst-font-size-base: 0.8rem;
```

Comment:
Thanks @gagb.
The one thing to keep in mind is that font size and whether a page scrolls is really dependent on the screen resolution of the viewer. On a low-res device, even reduced font sizes will still look large, and require scrolling. 
Also, small resolution may lead to accessibility scan errors down the line. 

For reference, [WCAG 2.1 recommends 18px](https://docs.merkulov.design/font-size-should-be-readable/#:~:text=WCAG%202.1%20Level%20AAA,required%20for%20WCAG%20AA%20compliance.)

0.8rem * 16px (browser default) = 12.8px
This maybe too small for comfortable reading (and trip accessibility scans). The recommended minimum is 16px (1rem).

Reply:
Okay, let's ignore this PR then! Thanks appreciate the feedback.